build

```diff
@@ -21,6 +21,8 @@ try {
 }
 
 shell('npx sucrase -q ./sim -d ./.sim-dist --transforms typescript,imports');
+shell('npx sucrase -q ./lib -d ./.lib-dist --transforms typescript,imports');
+shell('npx replace --silent \'(require\\\(.*?)(lib)(.*?\\\))\' \'$1.lib-dist$3\' .sim-dist/*');
```

Comment:
Don't we have a ripgrep dependency? That might have better perf.

Reply:
We don't, but you're right that `server/modlog` does seem to use it.